site/content/docs/user/configuration.md

```diff
@@ -179,6 +179,12 @@ The following options are available for setting on each entry in `nodes`.
 
 NOTE: not all options are documented yet!  We will fix this with time, PRs welcome!
 
+### Extra Node Labels
```

Comment:
this isn't really a config option of kind though, this is a kubeadm config patch.

we need to add a section on this, and probably add some higher level options in KIND